script/sync_prod_to_gcp/submissions_to_gcp.py

```diff
@@ -0,0 +1,928 @@
+"""
+submissions_to_gcp.py is an app that gets the published arxiv ID from the pub/sub queue on GCP,
+check the files exist on CIT and uploads them to GCP bucket.
+
```

Comment:
Please add something like:

"For each announced paper ID it will:
sync the /ftp files and ensure only current version files exist 
sync the abs and source files to /orig if needed
If TeX source it will build the PDF on a CIT web node and sync those to ps_cache
If HTML source it will build the HTML and sync those to the ps_cache"